Drivers/Src/MotorController.c

```diff
@@ -11,13 +13,71 @@
 #define MASK_CC_ERR 1<<2 //checks velocity on 2 bit
 #define MASK_OVER_SPEED_ERR 1<<24 //check if motor overshot max RPM on 24 bit
 
+static OS_SEM	MotorController_MailSem4;
+static OS_SEM	MotorController_ReceiveSem4;
+static float CurrentVelocity;
```

Comment:
We should initialize the CurrentVelocity to 0, because if we try to access it before a MotorController_Read gets a MOTOR_DRIVE message and updates it, we'll have undefined behavior